SafeSnap-Agents/AnswersSubmission/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "answers-submission-agent",
+  "version": "0.0.1",
+  "description": "Agent that detects answers events on the oracle for reality module questions.",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
+  },
+  "dependencies": {
+    "forta-agent": "^0.0.38",
+    "forta-agent-tools": "^1.0.34"
+    
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/nodemon": "^1.19.0",
+    "jest": "^27.0.6",
+    "nodemon": "^2.0.8",
+    "ts-jest": "^27.0.3",
+    "typescript": "^4.3.4",
+    "jest-when": "^3.5.1"
+  }
```

Comment:
I guess you should add `@types/jest-when` also here

Reply:
jest-when is no longer needed after using the `initialize` handler